CI/update_docker.sh

```diff
@@ -17,8 +17,7 @@ done
 
 ubuntu_versions="16.04 18.04"
 compilers="gcc clang"
-hdf5_versions="1.10.4"
```

Comment:
see here @gonuke
 we only builded for 1.10.4
and in CI we only test for 1.10.4 as well (we had the capability to test more tho)